docs/paper.md

```diff
@@ -0,0 +1,94 @@
+
+The software repository can be found at [https://github.com/nci/scores/](https://github.com/nci/scores/).
+
+# Statement of Need
+
+The research purpose of this software is (a) to mathematically verify and validate scientific research and (b) to foster research into new scores and metrics.
+
+## Key Benefits
+
+In order to meet the needs of researchers, `scores`:
+
+- is designed to work with n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. It has proper treatments for missing data, masking of data and weighting of results.
+- includes novel scores not commonly found elsewhere (e.g. FIRM, Flip-Flop Index).
+- is designed to work effectively with the libraries, data structures and methods commonly used in the meteorology, weather and climate communities. Scores can effectively handle the dimensionality, data size and data structures commonly utilised for: 
+  - gridded earth system data (e.g. Numerical Weather Prediction models) 
+  - tabular, point, lat/lon or site-based data (e.g. forecasts for specific locations)
+- includes a companion Jupyter Notebook for each score, metric and statistical test to demonstrate its use in practice
+- is highly modular and avoids extensive dependencies by providing its own implementations where relevant.
+- is intended to be easy to integrate and utilise in a wide variety of environments. It has been tested and used on workstations, servers and in high performance computing (supercomputing) environments. 
+- utilises Dask for scaling and performance
+
+## Roadmap and Future Development
+
+At the time of writing, the scores contained in this package are: Mean Squared Error (MSE), Mean Absolute Error (MAE), Root Mean Squared Error (RMSE), the Fixed Risk Multicategorical (FIRM) score [@Taggart:2022a], Continuous Ranked Probability Score (CRPS) for Cumulative Distribution Functions (CDFs) (including threshold-weighting, see [@Gneiting:2011]), CRPS for ensembles [@Gneiting_2007; @Ferro_2013], the Flip-Flop Index [@Griffiths:2019; @griffiths2021circular], Receiver Operating Characteristic (ROC) curves, the quantile score, and the Murphy score [@Ehm:2016]. It also includes the Diebold-Mariano statistical test [@Diebold:1995] with both the [@Harvey:1997] and [@Hering:2011] modifications. Additionally it contains isotonic regression which is becoming an increasingly important tool in forecast verification and can be used to generate stable reliability diagrams [@dimitriadis2021stable]. We anticipate more scores, metrics and statistical techniques will be added over time.
+
+The `scores` roadmap includes:
+
+- The addition of more scores, metrics and statistical techniques
+- Further optimisation and performance improvements
+- Increased support for machine learning library integration
+- Additional notebooks exploring complex use cases in depth
```

Comment:
And "increase pandas support"?

Reply:
In latest MR, the roadmap reference and dot points were removed.